src/Features/Core/Portable/SymbolSearch/Windows/SymbolSearchUpdateEngine.RemoteControlService.cs

```diff
@@ -2,24 +2,9 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#nullable disable
+namespace Microsoft.CodeAnalysis.SymbolSearch;
 
-using Microsoft.VisualStudio.RemoteControl;
-
-namespace Microsoft.CodeAnalysis.SymbolSearch
+internal partial class SymbolSearchUpdateEngine
```

Comment:
can you add a comment about why we need a blank internal partial here?

Reply:
Should have been deleted.